rust/auction/program/src/processor.rs

```diff
@@ -350,33 +360,37 @@ impl AuctionData {
         };
     }
 
-    pub fn is_winner(&self, key: &Pubkey) -> Option<usize> {
-        let minimum = match self.price_floor {
+    /// TODO: Provide implementation for methods below according to new bids count refactoring.
```

Comment:
Is this comment relevant or can be removed ?

Reply:
> Is this comment relevant or can be removed ?

Can be removed

Reply:
Then remove please 🙂